src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,97 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size >= table.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
 
+        Node node = new Node(key, value);
+        if (table[indexFor(key)] == null) {
```

Comment:
we call method `indexFor(key)` three times, let's create variable and write here result of calling this method